src/style-spec/rollup.config.js

```diff
@@ -6,12 +6,20 @@ import unassert from 'rollup-plugin-unassert';
 import json from 'rollup-plugin-json';
 import {flow} from '../../build/rollup_plugins';
 
+// Build es modules?
+const esm = 'esm' in process.env;
+
+const transforms = {dangerousForOf: true};
+if (esm) {
+    transforms.modules = false;
```

Comment:
could you add this param using ternary syntax in the object declaration itself? just for consistency.

Reply:
Thanks for the review @arindam1993, I have made the change you requested.